README.md

```diff
@@ -11,24 +11,26 @@ protected void Warning() {
 ```
 
 ## The most important thing
-I trust and expect my teams to do good work without me digging into the details, but I can and will dig all the way to bedrock when you or I need to.
+I trust and expect my teams to do good work without me digging into the details, but I can and will dig when you or I need to. I expect you to be the experts.
 
 ## What you can expect from me
 
 ### My working time
-I tend to work from 9:30-5:30 Eastern. I check slack and email earlier in the morning on my phone but it is rare that I will respond then. I endeavor to log out at 5:30 every day to be with my family and will not check slack or email after that time. For emergencies you can text me.
+I tend to work from 9:30-5:30 Eastern. I check slack and email earlier in the morning on my phone but it is rare that I will respond then and I definitely don't expect responses if I do. I log out at 5:30 every day to be with my family and will not check slack or email after that time. For emergencies you can text me. Because I don't check slack or email outside of my own decision to be working, you should feel totally safe to send me something at any time and I will read it when I'm back online.
 
-When I go on vacation, I unplug. I assign members of my team to cover various aspects of my role if the vacation is more than 2 days - for 2 or fewer days I assume everything can wait. If something catches fire you can text me, but please try not to catch any fires when I'm gone.
+During the COVID-19 pandemic, I use Care Time primarily to spend time with my daughter to help her with schoolwork or make sure she has something interesting to do. I may also use Care Time for groceries or to just take a walk.
 
-### 1:1s
-I will never cancel a 1:1 that can be rescheduled. I will add additional 1:1s if you ask for them. I am always available for pings of "hey can we chat for a minte?"
+When I go on vacation, I unplug. I assign members of my team to cover various aspects of my role if the vacation is more than 2 days - for 2 or fewer days I assume everything can wait. If something catches fire you can text me, but please try not to light any fires when I'm gone.
+
+I encourage you to own your calendar and your separation of work and home life. Block off commute time (yes even during the pandemic, my calendar now labels that "Family Time"). Take Care Time. Unplug when on vacation.
 
-If we are not co-located, I will book a conference room for you. If we are co-located I will prefer face-to-face and will usually move the 1:1 if one of us is working from home on the previously scheduled time.
+### 1:1s
+I will never cancel a 1:1 that can be rescheduled. I will add additional 1:1s if you ask for them. I will always respond to pings of "hey can we chat for a minte?" and will find the time. Please let me schedule the 1:1 so I can reschedule when necessary.
```

Comment:
Shouldn't this be:

```suggestion
I will never cancel a 1:1 that cannot be rescheduled. I will add additional 1:1s if you ask for them. I will always respond to pings of "hey can we chat for a minte?" and will find the time. Please let me schedule the 1:1 so I can reschedule when necessary.
```

Reply:
I'm pretty sure I thought I meant what I wrote. Maybe I should rethink the whole sentence with fewer negatives.

"I will always try to reschedule your 1:1 rather than cancel it if I have conflicts."